### DISCUSSION TITLE
Busco Equipo, o alguien que me quiera adoptar 🐄

### DISCUSSION THREAD

**Discussion by NoirOrange** [Launch-X-Latam/MisionFrontEnd#579] [Team Posts]
Buen día a todos! Busco un equipo para hackaton. 
Les dejo mi correo para que me escriban y podamos quedarnos de acuerdo 😊.

hayleyingx@gmail.com

**Comment by JenniDiaz1102** [comment]
Hola busco equipo para el hackaton... les dejo mi correo lx09274@innovaccion.mx

**Comment by luislordz** [comment]
Yo tambien, armalo y vamos agregando mas que esten en las mismas  Enviado desde Correo para Windows De: JenniDiaz1102Enviado: miércoles, 23 de noviembre de 2022 08:40 p. m.CC: SubscribedAsunto: Re: [Launch-X-Latam/mision-backend] Busco Equipo, o alguien que me quiera adoptar 🐄 (#118) Hola busco equipo para el hackaton... les dejo mi correo ***@***.***—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***> Libre de virus.www.avast.com

**Comment by NoirOrange** [comment]
@luislordz Plis, enviame un correo para irlo checando

**Comment by JuanCarlosCS19** [comment]
Hola Chicos, busco equipo para la hackaton :D mi correo es lx10495@innovaccion.mx

**Comment by luislordz** [comment]
Es ***@***.***  Enviado desde Correo para Windows De: NoirOrangeEnviado: miércoles, 23 de noviembre de 2022 10:58 p. m.CC: luislordz; MentionAsunto: Re: [Launch-X-Latam/mision-backend] Busco Equipo, o alguien que me quiera adoptar 🐄 ***@***.*** Plis, enviame un correo para irlo checando—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***> Libre de virus.www.avast.com
